src/inverseRobot.js

```diff
@@ -7,7 +7,20 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const swappedObj = {};
+  const seenValues = new Set();
+
+  for (const key in robot) {
+    const value = robot[key];
+
+    if (seenValues.has(value)) {
```

Comment:
Use 'swappedObj.hasOwnProperty(value)' instead of 'seenValues.has(value)' for checking repeated values.